modules/examiner/php/editexaminer.class.inc

```diff
@@ -70,8 +70,8 @@ class EditExaminer extends \NDB_Form
             // Access is only granted if user sites are a superset of examiner sites
             // considering that certification changes affect all sites
             return $user->hasPermission('examiner_view')
-                && (($user->hasPermission('examiner_multisite')
-                || empty(array_diff($centerIDs, $user->getData('CenterIDs')))));
+                || $user->hasPermission('examiner_multisite')
```

Comment:
@MounaSafiHarab Is there any reason to have this permission at all now that users can be affiliated with multiple sites and not just use `$user->hasCenterPermission('examiner_view')`?

(That change is probably beyond the scope of 20.0.. but just in general?)

Reply:
I think this should be `(examiner_view || examiner_multisite) && empty(array_diff(...))` (ie require one of view or multisite), because if not this would return true even if the user didn't have examiner_view or multisite, as long as they belonged to all of the centers (unless this is what is intended).

Reply:
Yeah that makes sense. I'll change this line accordingly for now and @MounaSafiHarab can let us know if it's still necessary

Reply:
@driusan @johnsaigle 
I am not sure if examiner multisite and user multisite is supposed to be identical?
is an examiner always a user?
I really don't know.

@ridz1208 you made this initial change, can you please comment?

Reply:
@driusan @johnsaigle 

I personally feel that any `_multisite` permission is now antiquated (including `access_all_profiles`) which I brought up during the meeting after @MounaSafiHarab 's PR got merged... everyone seemed to disagree saying it's not the same as being affiliated with all the sites....

So I dont have an answer here

Reply:
@ridz1208 @MounaSafiHarab Thanks for weighing in here.

It sounds like we don't have a good reason to change this setting at this time so I feel like we shouldn't adjust it until we are sure that `_multisite` is deprecated.